packages/frontend/app/(dashboard)/course/[cid]/insights/components/InsightCard.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+type InsightCardProps = {
+  children: React.ReactNode
+  title: string
+  description?: string
+}
+
+const InsightCard: React.FC<InsightCardProps> = ({
+  children,
+  title,
+  description,
+}) => {
+  return (
+    <div className="border-b-helpmeblue-light hover:border-b-helpmeblue flex-[1_1_auto] rounded-lg border-b-2 bg-gray-50 p-8 drop-shadow-md transition-all">
```

Comment:
```suggestion
    <div className="border-b-helpmeblue-light hover:border-b-helpmeblue flex-auto rounded-lg border-b-2 bg-gray-50 p-8 drop-shadow-md transition-all">
```
flex-auto is the same as doing flex-[1_1_auto]

Reply:
addressed, made changes